ansible/configs/ocp4-cluster/destroy_env.yml

```diff
@@ -17,7 +17,7 @@
     - name: Remove DNS entry for OpenShift API and ingress
       nsupdate:
         server: "{{ osp_cluster_dns_server }}"
-        zone: "{{ cluster_dns_zone }}"
+        zone: "{{ osp_cluster_dns_zone }}"
```

Comment:
where are you setting this? I don't see it in default_vars?

Reply:
Secret vars

Reply:
Since it depends on the cluster